### PR TITLE
Use counts in todolist

### PR DESCRIPTION
**Pull request by gaetanHerve**
* [core] add missing tableName properties in model constructors
* [space] improve todolist sql resquests

### CODE REVIEW

File: Framework/Model.php [hunk 1]
```diff
@@ -407,6 +407,13 @@ public function admCount($tableName, $id_space= 0) {
         return $this->runRequest($sql, $params)->fetch();
     }
 
+    public function countForSpace($id_space) {
```
Comment (osallou) [inline Framework/Model.php:410]: Should do that in admcount with a if on id space

File: Modules/core/Model/CoreUser.php [hunk 2]
```diff
@@ -980,7 +980,18 @@ public function getSpaceActiveUsers($id_space) {
...
+    }
+
+    public function countSpaceActiveUsers($id_space) {
+        $sql = "SELECT count(core_users.id) AS total "
+                . "FROM core_j_spaces_user "
+                . "INNER JOIN core_users ON core_j_spaces_user.id_user = core_users.id "
+                . "WHERE core_j_spaces_user.id_space=?";
+        $req = $this->runRequest($sql, array($id_space));
+        $total = $req->fetch();
+        Configuration::getLogger()->debug("[TEST]", ["total" => $total]);
```
Comment (osallou) [inline Modules/core/Model/CoreUser.php:992]: Debug log

**Comment by osallou** [comment]
Pret pour merge? Modifs testées?

**Comment by gaetanHerve** [comment]
C'est ok !

**Pull request merged**